### PR TITLE
updating wake lock and clipboard adding content index and periodic sync

### PR DESCRIPTION
**Pull request by Rumyra**
Updates are:

- Added Screen Wake Lock to GroupData
- Updated Clipboard in GroupData to reflect landing page
- Added Content Index API
- Added Periodic Sync API

All tests pass my end

@chrisdavidmills 👍

### CODE REVIEW

File: macros/GroupData.json [hunk 1]
```diff
@@ -96,6 +97,14 @@
                             "Element: cut",
                             "Element: paste" ]
         },
+        "Content Index API": {
+            "overview":   [ "Content Index API" ],
+            "interfaces": [ "ContentIndex", "ContentIndexEvent" ],
```
Comment (chrisdavidmills) [inline macros/GroupData.json:102]: Put each interface on a single line, and line up the opening quote marks of each one.

File: macros/GroupData.json [hunk 2]
```diff
@@ -96,6 +97,14 @@
...
+            "methods":    [],
+            "properties": ["ServiceWorkerRegistration.index",
+                "ServiceWorkerGlobalScope.oncontentdelete"],
```
Comment (chrisdavidmills) [inline macros/GroupData.json:105]: Line up the opening quote marks.

File: macros/GroupData.json [hunk 3]
```diff
@@ -85,6 +85,7 @@
             "events":     []
         },
         "Clipboard API": {
+            "overview":   [ "Clipboard API" ],
             "interfaces": [ "Clipboard",
             "ClipboardEvent",
```
Comment (chrisdavidmills) [inline macros/GroupData.json:90]: Line up opening quotes.

File: macros/GroupData.json [hunk 4]
```diff
@@ -1018,6 +1036,13 @@
             "properties": [ "Screen.orientation"],
             "events":     []
         },
+        "Screen Wake Lock API": {
+            "overview":   [ "Screen Wake Lock API" ],
+            "interfaces": [ "WakeLock", "WakeLockSentinel", "Navigator.wakelock" ],
```
Comment (chrisdavidmills) [inline macros/GroupData.json:1041]: Put each interface on a separate line, and line up the opening quotes.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Just a few stylistic bits to update, but looking good overall!

File: macros/GroupData.json (hunk 4 above)
Comment (chrisdavidmills) [inline macros/GroupData.json:1041]: You've still got multiple interfaces on a single line here.

File: macros/GroupData.json (hunk 1 above)
Comment (chrisdavidmills) [inline macros/GroupData.json:102]: You've still got multiple interfaces on a single line here.

File: macros/GroupData.json (hunk 2 above)
Comment (chrisdavidmills) [inline macros/GroupData.json:105]: Needs lining up.

**Review by chrisdavidmills — COMMENTED** [review]
Couple more nitpicks.

**Comment by Rumyra** [comment]
Thanks @chrisdavidmills - I think i might have done something odd 🤷🏻‍♀️ these were the original requests which I amended. They should all be there now 👍

File: macros/GroupData.json [hunk 5]
```diff
@@ -1018,6 +1037,15 @@
             "properties": [ "Screen.orientation"],
             "events":     []
         },
+        "Screen Wake Lock API": {
+            "overview":   [ "Screen Wake Lock API" ],
+            "interfaces": [ "WakeLock",
+                            "WakeLockSentinel",
+                            "Navigator.wakelock" ],
```
Comment (chrisdavidmills) [inline macros/GroupData.json:1044]: Sorry, one last nitpick — `Navigator.wakelock` is a property, not an interface.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Heh, sorry, didn't notice this one last time around.

The rest of it looks fine now.

File: macros/GroupData.json (hunk 5 above)
Comment (Rumyra) [reply macros/GroupData.json]: Sorry of course it is :) should be amended

**Review by chrisdavidmills — APPROVED** [review]
OK, looks like we are good to go.

**Pull request merged**